File: node/views/getPontos.php
<?php
$serverName = 'regulus.cotuca.unicamp.br'; // Substitua pelo nome do servidor SQL Server
$connectionOptions = array(
    "Database" => 'BD23532', // Substitua pelo nome do banco de dados
    "Uid" => 'BD23532', // Substitua pelo nome de usuário
    "PWD" => 'BD23532' // Substitua pela senha
);

$conn = sqlsrv_connect($serverName, $connectionOptions);

if (!$conn) {
    die("Erro na conexão com o banco de dados: " . sqlsrv_errors());
}

$result = sqlsrv_query($conn, "SELECT idPonto, logradouro, lat, long FROM Pontos");

$data = array();
while ($row = sqlsrv_fetch_array($result, SQLSRV_FETCH_ASSOC)) {
    $data[] = $row;
}

sqlsrv_free_stmt($result);
sqlsrv_close($conn);

echo json_encode($data);
?>

File: node/views/getPontos.cs
using System;
using System.Collections.Generic;
using System.Data;
using MySql.Data.MySqlClient;

public List<Ponto> GetPontosDoBancoDeDados()
{
    string connectionString = "Server=regulus.cotuca.unicamp.br;Database=BD23532;Uid=BD23532;Pwd=BD23532;";

    List<Ponto> pontos = new List<Ponto>();

    using (MySqlConnection connection = new MySqlConnection(connectionString))
    {
        connection.Open();

        string sql = "SELECT idPonto, logradouro, lat, lon FROM lb_Pontos";

        using (MySqlCommand command = new MySqlCommand(sql, connection))
        {
            using (MySqlDataReader reader = command.ExecuteReader())
            {
                while (reader.Read())
                {
                    Ponto ponto = new Ponto
                    {
                        IdPonto = reader.GetInt32(0),
                        Logradouro = reader.GetString(1),
                        Latitude = reader.GetString(2),
                        Longitude = reader.GetString(3)
                    };

                    pontos.Add(ponto);
                }
            }
        }
    }

    return pontos;
}
